html-forms-homework/homework-order-form.html/excercise-2.html

```diff
@@ -0,0 +1,316 @@
+              <option value="MF">St Martin</option>
+              <option value="PM">St Pierre &amp; Miquelon</option>
+              <option value="VC">St Vincent &amp; Grenadines</option>
+              <option value="SR">Suriname</option>
+              <option value="SJ">Svalbard &amp; Jan Mayen</option>
+              <option value="SE">Sweden</option>
+              <option value="CH">Switzerland</option>
+              <option value="TW">Taiwan</option>
+              <option value="TJ">Tajikistan</option>
+              <option value="TZ">Tanzania</option>
+              <option value="TH">Thailand</option>
+              <option value="TL">Timor-Leste</option>
+              <option value="TG">Togo</option>
+              <option value="TK">Tokelau</option>
+              <option value="TO">Tonga</option>
+              <option value="TT">Trinidad &amp; Tobago</option>
+              <option value="TA">Tristan da Cunha</option>
+              <option value="TN">Tunisia</option>
+              <option value="TR">Turkey</option>
+              <option value="TM">Turkmenistan</option>
+              <option value="TC">Turks &amp; Caicos Islands</option>
+              <option value="TV">Tuvalu</option>
+              <option value="UG">Uganda</option>
+              <option value="UA">Ukraine</option>
+              <option value="AE">United Arab Emirates</option>
+              <option value="GB">United Kingdom</option>
+              <option value="US">United States</option>
+              <option value="UY">Uruguay</option>
+              <option value="UM">US Outlying Islands</option>
+              <option value="VI">US Virgin Islands</option>
+              <option value="UZ">Uzbekistan</option>
+              <option value="VU">Vanuatu</option>
+              <option value="VA">Vatican City</option>
+              <option value="VE">Venezuela</option>
+              <option value="VN">Vietnam</option>
+              <option value="WF">Wallis &amp; Futuna</option>
+              <option value="EH">Western Sahara</option>
+              <option value="YE">Yemen</option>
+              <option value="ZM">Zambia</option>
+              <option value="ZW">Zimbabwe</option>
+            </select>
+          </div>
+    </fieldset>
+    <fieldset style="width: 500px;">
+        <legend>Step 2</legend>
+        <label for="nameOnCard">Name on card *</label> <br>
+        <input type="text" name="nameOnCard" id="nameOnCard" required><br>
+        <label for="cardnumber">Card number *</label><br>
+        <input type="number" name="cardnumber" id="cardnumber" required pattern="[0-9]"><br>
+        <label for="expirydate">Expiry date *</label><br>
+        <input type="date" name="expirydate" id="expirydate" required minlength="4"pattern="[0-9]" ><br>
+        <label for="securitycode">Security code *</label><br>
+        <input type="number" name="securitycode" id="securitycode" required minlength="3" maxlength="4" pattern="[0-9]" ><br>
```

Comment:
The size attribute is not applied on the type="number" field. It's only applied to text fields.